priv/templates/phx.gen.html/controller_test.exs

```diff
@@ -12,59 +12,78 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
     <%= schema.singular %>
   end
 
-  test "lists all entries on index", %{conn: conn} do
-    conn = get conn, <%= schema.route_helper %>_path(conn, :index)
-    assert html_response(conn, 200) =~ "Listing <%= schema.human_plural %>"
+  describe "index" do
+    test "lists all entries on index", %{conn: conn} do
+      conn = get conn, <%= schema.route_helper %>_path(conn, :index)
+      assert html_response(conn, 200) =~ "Listing <%= schema.human_plural %>"
+    end
   end
 
-  test "renders form for new <%= schema.plural %>", %{conn: conn} do
-    conn = get conn, <%= schema.route_helper %>_path(conn, :new)
-    assert html_response(conn, 200) =~ "New <%= schema.human_singular %>"
+  describe "new" do
```

Comment:
The intention here was to keep them matching the action names. Am happy to append the resource name though. I'll update the PR.